geometry_msgs/Pose2D input
---
geometry_msgs/Pose2D output
